### PR TITLE
[fix] Refactoring to remove warnings

### PR DESCRIPTION
**Pull request by JavaEria**
image

### CODE REVIEW

**Comment by JavaEria** [comment]
@abdulmoizeng Please review

File: client/src/containers/CurrentEvent/CurrentEvent.js [hunk 1]
```diff
@@ -19,20 +17,14 @@ import { fetchCurrentEvent } from '../../actions';
...
-import EditUserProfile from '../EditProfile/EditUserProfile';
 import { Link } from 'react-router-dom';
 import { AttendeeType } from '../../types/attendee-types';
 import './CurrentEvent.css';
 
 const DATE_FORMAT = 'LLLL';
 
-class CurrentEvent extends Component {
+class CurrentEvent extends Component<Props> {
   eventId: string;
```
Comment (oahmedqureshi) [inline client/src/containers/CurrentEvent/CurrentEvent.js]: @taleymirza I don't think we need to define eventId type here?

File: client/src/containers/ForgotPassword/ForgotPassword.js [hunk 2]
```diff
@@ -14,9 +14,9 @@ type Props = BaseReduxPropTypes & {
 };
 
 class ForgotPassword extends Component<Props> {
-  constructor(props) {
-    super(props);
-  }
+  // constructor(props) {
```
Comment (oahmedqureshi) [inline client/src/containers/ForgotPassword/ForgotPassword.js]: Please remove commented code

File: client/src/containers/CurrentEvent/CurrentEvent.js (hunk 1 above)
Comment (taleymirza) [reply client/src/containers/CurrentEvent/CurrentEvent.js]: @oahmedqureshi I am not working on this PR, @JavaEria! please review

File: client/src/containers/ForgotPassword/ForgotPassword.js (hunk 2 above)
Comment (mashhoodr) [reply client/src/containers/ForgotPassword/ForgotPassword.js]: @JavaEria this isnt fixed yet

**Pull request merged**